projects/angular/src/data/datagrid/API.md

```diff
@@ -0,0 +1,53 @@
+# Datagrid - Style API
+
+## CSS Variables
+
+| CSS Variable Name                                     | Description                                                                  |
+| ----------------------------------------------------- | -----------------------------------------------------------------------------|
+| --clr-datagrid-font-color                             | Changes the text color of datagrid.                                          |
+| --clr-datagrid-default-border-color                   | Changes the border color of datagrid when default.                           |
+| --clr-datagrid-icon-color                             | Changes the header icons color of datagrid.                                  |
+| --clr-datagrid-row-hover-color                        | Changes the row background color of the data table when hovers over element. |
+| --clr-datagrid-row-hover-font-color                   | Changes the row text color of data table when hovers over element.           |
+| --clr-datagrid-action-toggle-color                    | Changes the toggle action button color.                                      |
+| --clr-datagrid-pagination-btn-color                   | Changes the table bottom pagination buttons color.                           |
+| --clr-datagrid-pagination-btn-disabled-color          | Changes the table bottom pagination buttons color when disabled.             |
+| --clr-datagrid-pagination-input-border-color          | Changes the text box border color of table datagrid.                         |
+| --clr-datagrid-pagination-input-border-focus-color    | Changes the text box border color of table datagrid when focus on element.   |
+| --clr-datagrid-popover-bg-color                       | Changes the background color of popup box.                                   |
+| --clr-datagrid-popover-border-color                   | Changes the border color of popup box.                                       |
+| --clr-datagrid-row-selected                           | Changes the text color of table row when user selected.                      |
+| --clr-datagrid-loading-background                     | Changes the background color of datagrid when click on toggle spinner.       |
+| --clr-datagrid-popovers-box-shadow-color              | Changes the popup box shadow color of datagrid.                              |
+| --clr-datagrid-column-switch-header-font-color        | Changes the column header text color of datagrid.                            |
+| --clr-datagrid-column-switch-header-font-hover-color  | Changes the column header text color of datagrid when hovers over element.   |
+| --clr-datagrid-detail-caret-icon-open-bg-color        | Changes the caret icon background color of datagrid.                         |
```

Comment:
Changes the datagrid open detail caret icon background color.

Reply:
Changed as per your suggestion.